foreman/foreman.spec

```diff
@@ -588,6 +584,7 @@ Requires: %{?scl_prefix}rubygem(jquery_pwstrength_bootstrap_4) >= 1.2
 Requires: %{?scl_prefix}rubygem(jquery_pwstrength_bootstrap_4) < 2.0
 Requires: %{?scl_prefix}rubygem(jquery-turbolinks) >= 2.1
 Requires: %{?scl_prefix}rubygem(jquery-turbolinks) < 3.0
+Obsoletes: npm(flux)
```

Comment:
I don't think this is location correct, it should be `Obsoletes: npm(flux) < 2.1.1-2` on `tfm/tfm.spec` - the foreman package does not obsolete this.